### PR TITLE
Profile creation pages: layout/validation/translation improvements

### PR DESCRIPTION
**Pull request by myxmaster**
- Harmonized button styles
- Improved checkbox style
- Improved terms and conditions checkbox text
- Improved mnemonic input validation (show error even if first word is wrong)
- Removed unused translation and notification types
- Minor layout improvements

grafik

### CODE REVIEW

File: src/language/zh-CN.json [hunk 1]
```diff
@@ -608,7 +608,7 @@
     "mnemonic_copy": "复制",
     "mnemonic_generation_error": "生成钱包种子失败",
     "mnemonic_inconsistency": "助记词不一致",
-    "mnemonic_input_error": "助记词输入错误",
+    "mnemonic_correct": "记忆法是正确的",
```
Comment (rg911) [inline src/language/zh-CN.json:611]: "mnemonic_correct": "记忆法是正确的" => "mnemonic_correct": "助记词正确"

**Review by rg911 — CHANGES_REQUESTED** [review]
Please resolve conflict 
👍

File: src/language/zh-CN.json (hunk 1 above)
Comment (myxmaster) [reply src/language/zh-CN.json]: done

**Comment by myxmaster** [comment]
conflicts resolved

**Review by rg911 — APPROVED** [review]

**Pull request merged**